Src/BootCamp.Chapter/FileLogger.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    class FileLogger : ILogger
+    {
+        readonly string _file;
+        public FileLogger(string file)
+        {
+            _file = file;
+        }
+
+        public void LogStart()
+        {
+            File.AppendAllText(_file, Environment.NewLine);
+            File.AppendAllText(_file, $"{System.DateTime.Now} - Program start{Environment.NewLine}");
+        }
+
+        public void LogExit()
+        {
+            File.AppendAllText(_file, $"{System.DateTime.Now} - Program exit{Environment.NewLine}");
+        }
+
+        public void Error(Exception ex)
```

Comment:
Special method for logging error is good, but you should name it a  verb. You log an error so name it LogError.